app/dashboard/models.py

```diff
@@ -920,8 +918,6 @@ def value_in_usdt_then(self):
         decimals = 1
         if self.tokenName == 'USDT':
             return float(self.amount)
-        if self.tokenName == 'DAI':
-            return float(self.amount / 10 ** 18)
```

Comment:
instead of removing these lines can we make them say

`float(self.amount)`

instead?